service/internal/db/db.go

```diff
@@ -91,3 +106,29 @@ func (d SQLClient) DeleteProjectEntry(ctx context.Context, project string) error
 
 	return sess.WithContext(ctx).Collection(ProjectEntryDB).Find("project", project).Delete()
 }
+
+func (d SQLClient) CreateTokenEntry(ctx context.Context, project string, secret_id_accessor string) (TokenEntry, error) {
+	res := TokenEntry{}
+
+	sess, err := d.createSession()
+	if err != nil {
+		return res, err
+	}
+	defer sess.Close()
+
+	err = sess.WithContext(ctx).Tx(func(sess db.Session) error {
+		res = TokenEntry{
+			CreatedAt: time.Now().Format(time.RFC3339),
```

Comment:
I believe this is in local time. Should we be using UTC here?

Reply:
sure thing this can be in UTC.